--- elogind-239.1/src/basic/selinux-util.c.orig	2018-10-17 19:50:46.000000000 +0200
+++ elogind-239.1/src/basic/selinux-util.c	2018-11-22 05:40:37.017117099 +0100
@@ -16,12 +16,12 @@
 #endif
 
 #include "alloc-util.h"
-//#include "fd-util.h"
+#include "fd-util.h"
 #include "log.h"
 #include "macro.h"
 #include "path-util.h"
 #include "selinux-util.h"
-//#include "stdio-util.h"
+#include "stdio-util.h"
 #include "time-util.h"
 #include "util.h"
 
